### PR TITLE
Add Landing Page + Request Access

### PR DESCRIPTION
**Pull request by hmillison**
Screenshot 2023-06-21 at 14 51 49

### CODE REVIEW

File: src/components/SolarSystem/OrbitCanvas.tsx [hunk 1]
```diff
@@ -0,0 +1,350 @@
...
+          ctx.arc(circleX, circleY, config.size, 0, Math.PI * 2);
+          ctx.fill();
+    
+          config.angle += config.speed;
+    
+          ctx.restore();
+        };
+    
+        ctx.clearRect(0, 0, canvas.width, canvas.height);
+        (ctx as any).reset();
```
Comment (bolandrm) [inline src/components/SolarSystem/OrbitCanvas.tsx:318]:
```suggestion
        (ctx as any).restore();
```

**Comment by ryw** [comment]
Love you work @hmillison 

A few things I see:

- [ ] on docs/blogs, the content scrolls through the header
- [ ] when mouse hovering over "hello postgres" the top section has a bit of scroll (on Safari/Mac at least) before the whole window scrolls, before the button floats down and to the left — I can record screencast of it if you can't see it - doesn't happen if hovering over right side of screen
- [ ] "Join us" at bottom would be clearer if we say "Join the team" — I saw it as another "waitlist" link at first
- [ ] Would be great if we could unify the header/footer from the homepage into the blog + docs rather than having 2 styles
- [ ] tool left logo text hover is pink - maybe a leftover from our other branding? not sure what it should be
- [ ]

**Comment by hmillison** [comment]
thanks for the feedback @ryw - will send over to the PM so she can prioritize those changes

**Comment by ryw** [comment]
One other thing @hmillison - on mobile, the page can be pulled left/right as you scroll - the docs + blog pages don't behave that way - so maybe needs some css tweak

**Comment by ryw** [comment]
"Hello Postgres" needs something like `letter-space: -0.5rem;` :)

**Comment by sjmiller609** [comment]
the social card could be updated FYI, you can see that when you text yourself the preview environment link

**Comment by sjmiller609** [comment]
I noticed if you spend time on mobile looking at the site, for example zoom in and try to read stuff, scroll up and down, it starts to have some issues. It’s looking good, hope you don’t mind the feedback I know it’s a work in progress!

**Comment by sjmiller609** [comment]
🎉

**Pull request merged**